src/__screenshot_tests__/tooltip-screenshot-test.tsx

```diff
@@ -107,3 +108,16 @@ test('Tooltip - active tooltip is closed if another one is opened', async () =>
     const thirdTooltip = await page.screenshot();
     expect(thirdTooltip).toMatchImageSnapshot();
 });
+
+test('Tooltip - arrow appears properly in Vivo_New skin when target is close to viewport edges', async () => {
```

Comment:
Please, add a comment explaining why the "in Vivo_New skin" part is relevant (I guess it's because of the border radius)